src/haz3lcore/zipper/Printer.re

```diff
@@ -60,69 +60,43 @@ let to_rows =
   };
 };
 
-let pretty_print = (~measured: Measured.t, z: Zipper.t): string =>
+let pretty_print =
+    (~holes: option(string)=Some(""), ~measured: Measured.t, z: Zipper.t)
```

Comment:
Seems like this doesn't need to be optional anymore

Reply:
perhaps we can agree on a canonical textual representation for holes

Reply:
⦿